app/main/forms/auth_forms.py

```diff
@@ -62,6 +62,15 @@ class CreateUserForm(Form):
         ]
     )
 
+    phone_number = StringField(
+        'Phone number', id="input_phone_number",
+        validators=[
+            Regexp("^$|^\\+?([\\d\\s()-]){6,20}$",
+                   message="Phone numbers must be between 6 and 20 characters."
```

Comment:
Have talked to Sarah and Chris. We're going to update to this message for now: 

'Phone numbers must be at least 7 characters long. They can only include digits, spaces, plus and minus signs, and brackets.'

But will work on a separate story to make the validation active and consistent on this page and elsewhere in the app.